src/Integration.Vsix/Commands/ProjectSonarLintMenuCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ProjectSonarLintMenuCommand.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using EnvDTE;
+using Microsoft.VisualStudio.Shell;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace SonarLint.VisualStudio.Integration.Vsix
+{
+    internal class ProjectSonarLintMenuCommand : VsCommandBase
```

Comment:
Required to work around issues with hiding menus where all children are hidden/disabled.